blogs/2016/05/04/extension-roundup-may.md

```diff
@@ -16,14 +16,6 @@ The Visual Studio Code Extension [Marketplace](https://marketplace.visualstudio.
 
 If you are new to VS Code extensions, check out the [docs](/docs/editor/extension-marketplace.md) for instructions on finding and installing new extensions.
 
-## Debugger for Edge by Microsoft JS Diagnostics
```

Comment:
Do we need to remove it from a 6 year old blog post? Feels like more of a historical record than an up to date reference